benchmark/runner.py

```diff
@@ -289,6 +289,7 @@ def run_docker(definition, dataset, count, runs, timeout, rebuild,
             },
             cpuset_cpus=cpu_limit,
             mem_limit=mem_limit,
+            privileged=True, # so that we can run perf inside the container
```

Comment:
can be reverted as mentioned in PR descr, only useful for FlameGraph'ing purposes